src/js/utils/index.d.ts

```diff
@@ -193,6 +200,7 @@ export type BorderType =
       side?: BoxSideType;
       size?: BoxSizeType;
       style?: BoxStyleType;
+      image?: string | { source?: string; slice?: number };
```

Comment:
Wondering about the straight "string" support. We don't tend to support CSS shorthands directly on Grommet props. For example, I couldn't do: <Box border="1px dashed red" />

Should we just support the object for now?

Reply:
yeah I can change that.. good point

Reply:
@jcfilben What do you think?

Reply:
Yeah I think we could just support object for now, @taysea your right we don't normally include shorthand

Reply:
done!